cfgov/prepaid_agreements/jinja2/prepaid_agreements/index.html

```diff
@@ -69,11 +69,12 @@ <h3 class="h4 u-mb5">Search within</h3>
                     </div>
                     <div class="flex-all">
                         <div class="m-btn-inside-input input-contains-label">
-                            <label for="searchText" class="input-contains-label_before input-contains-label_before__search">
+                            <label for="searchText"
+                                   class="input-contains-label_before input-contains-label_before__search"
+                                   aria-label="The term to search for">
```

Comment:
I think in this case you would apply the `aria-label` to the `input`, not the `label`.

Reply:
I don't think so?

> The aria-label attribute is used to define a string that labels the current element. Use it in cases where a text label is not visible on the screen.
([ref](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute))

This label only contains an icon, so this `aria-label` is labeling that icon.

Reply:
Thinking about this further, I think you're right. We'd `aria-label` the `input` and `aria-hidden="true"` the `label` in case that would be read.